apiserver/resources_unit_test.go

```diff
@@ -149,3 +150,11 @@ func (s *stubResourceOpener) OpenResource(_ context.Context, name string) (resou
 	}
 	return s.ReturnOpenResource, nil
 }
+
+func (s *stubResourceOpener) SetResourceUsed(_ context.Context, name string) error {
```

Comment:
This suite is small. I'd rather convert it to mocks that keep adding to the stub infra.

Reply:
Done.